### DISCUSSION TITLE
Way to enable bing search in search bar

### DISCUSSION THREAD

**Discussion by DikshitRJ** [Raphire/Win11Debloat#112] [Q&A]
Using this tool, I disabled bing search in windows search bar by mistake. however I want that feature back again. I am unable to do so. Could you please guide me to re-enable bing search in search bar?

**Comment by Raphire** [comment]
Heya,

Re-enabling bing web search is luckily quite easy as the script provides registry files to undo specific tweaks. 

In your case you need to download and run [this regfile](https://github.com/Raphire/Win11Debloat/blob/d785704e0811b907f01847a5d995f2fd0f175a0f/Regfiles/Undo/Enable_Bing_Cortana_In_Search.reg). After this you may need to sign-out or restart your pc to apply the changes. 

If your Windows region is set to a country within the EEA you may also need to install the Bing Web Search app [here](https://apps.microsoft.com/detail/9nzbf4gt040c) and enable it in the Windows Search settings.

Hope this helps!
